_includes/analytics-gain.html

```diff
@@ -8,17 +8,17 @@ <h4 class="font-weight-light">
                 <p>Grandes empresas ya se benefician de datos obtenidos de
                     <span class="analytics-brand">Frogtek Analytics</span></p>
             </h4>
-            <p class="display-4">¡no te quedes atrás!</p>
+            <p class="display-4">¡No te quedes atrás!</p>
             <ul class="text-left d-inline-block ">
                 <li>Aprende del mercado en tiempo real</li>
                 <li>Consigue conclusiones accionables</li>
                 <li>Mejora la efectividad en ventas</li>
             </ul>
             <div>
-                <a href="{{ site.analytics_contact_url }}" class="btn btn-lg btn-primary"
-                   data-aos="zoom-out" data-aos-easing="ease-in-sine" data-aos-duration="1000"
-                >contacta con nosotros</a>
+                <a href="{{ site.analytics_form_url }}" class="btn btn-lg btn-primary"
+                   data-aos="zoom-out" data-aos-easing="ease-in-sine" data-aos-duration="1000" target="_blank">
```

Comment:
This is widely considered a very bad practice... You are forcing the user to open a tab when the decision should be his